.github/workflows/unit_tests.yml

```diff
@@ -47,59 +47,18 @@ on:
 jobs:
   unit-tests:
     name: Unit tests
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        # We are specifying only the major and minor of the docker images to automatically pick up the latest patch release
-        swift:
-          - image: "swift:5.9-jammy"
-            swift_version: "5.9"
-            enabled: ${{ inputs.linux_5_9_enabled }}
-          - image: "swift:5.10-jammy"
-            swift_version: "5.10"
-            enabled: ${{ inputs.linux_5_10_enabled }}
-          - image: "swift:6.0-jammy"
-            swift_version: "6.0"
-            enabled: ${{ inputs.linux_6_0_enabled }}
-          - image: "swiftlang/swift:nightly-6.0-jammy"
-            swift_version: "nightly-6.0"
-            enabled: ${{ inputs.linux_nightly_6_0_enabled }}
-          - image: "swiftlang/swift:nightly-main-jammy"
-            swift_version: "nightly-main"
-            enabled: ${{ inputs.linux_nightly_main_enabled }}
-    steps:
-      - name: Checkout repository
-        if: ${{ matrix.swift.enabled }}
-        uses: actions/checkout@v4
-        with:
-          persist-credentials: false
-          submodules: true
-      - name: Mark the workspace as safe
-        if: ${{ matrix.swift.enabled }}
-        # https://github.com/actions/checkout/issues/766
-        run: git config --global --add safe.directory ${GITHUB_WORKSPACE}
-      - name: Run matrix job
-        if: ${{ matrix.swift.enabled }}
-        env:
-          SWIFT_VERSION: ${{ matrix.swift.swift_version }}
-          COMMAND: "swift test"
-          COMMAND_OVERRIDE_5_9: "swift test ${{ inputs.linux_5_9_arguments_override }}"
-          COMMAND_OVERRIDE_5_10: "swift test ${{ inputs.linux_5_10_arguments_override }}"
-          COMMAND_OVERRIDE_6_0: "swift test ${{ inputs.linux_6_0_arguments_override }}"
-          COMMAND_OVERRIDE_NIGHTLY_6_0: "swift test ${{ inputs.linux_nightly_6_0_arguments_override }}"
-          COMMAND_OVERRIDE_NIGHTLY_MAIN: "swift test ${{ inputs.linux_nightly_main_arguments_override }}"
-        run: |
-          apt-get -qq update && apt-get -qq -y install curl && apt-get -y install libsasl2-dev
-          curl -s https://raw.githubusercontent.com/apple/swift-nio/main/scripts/check-matrix-job.sh | bash
-    container:
-      image: ${{ matrix.swift.image }}
-    services:
-      zookeeper:
-        image: ubuntu/zookeeper
-      kafka:
-        image: ubuntu/kafka
-        env:
-          ZOOKEEPER_HOST: zookeeper
-    env:
-      KAFKA_HOST: kafka
+    # Workaround https://github.com/nektos/act/issues/1875
```

Comment:
```suggestion
```

This comment is not relevant here, it's to do with chained workflow imports, it's copy-pasta.